src/engine/localevent.cpp

```diff
@@ -927,29 +940,29 @@ namespace EventProcessing
 
 #if defined( TARGET_NINTENDO_SWITCH )
             // Custom button mapping for Nintendo Switch
-            if ( buttonType == LocalEvent::CONTROLLER_BUTTON_A ) {
-                buttonType = LocalEvent::CONTROLLER_BUTTON_B;
+            if ( buttonType == LocalEvent::ControllerButtonType::CONTROLLER_BUTTON_A ) {
+                buttonType = LocalEvent::ControllerButtonType::CONTROLLER_BUTTON_B;
             }
-            if ( buttonType == LocalEvent::CONTROLLER_BUTTON_B ) {
-                buttonType = LocalEvent::CONTROLLER_BUTTON_A;
+            if ( buttonType == LocalEvent::ControllerButtonType::CONTROLLER_BUTTON_B ) {
```

Comment:
Hi, @ihhub to fix Nintendo Switch gamepad buttons remap we should fix this line:
```suggestion
            else if ( buttonType == LocalEvent::ControllerButtonType::CONTROLLER_BUTTON_B ) {
```

Reply:
Thanks for pointing out!